code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -143,20 +143,12 @@ Credit where due:
 		greet_servant(L)
 		equip_servant(L)
 		add_servant_of_ratvar(L, TRUE)
-	addtimer(CALLBACK(src, .proc/unlock_application_scripture), (ark_time * 0.5) * 600)
-	addtimer(CALLBACK(src, .proc/cry_havoc), ark_time * 600) //600 deciseconds in a minute * number of minutes
+	var/obj/structure/destructible/clockwork/massive/celestial_gateway/G = GLOB.ark_of_the_clockwork_justiciar //that's a mouthful
+	G.initial_activation_delay = ark_time * 60
```

Comment:
if `GLOB.ark_of_the_clockwork_justiciar` was properly cast you could just do `GLOB.ark_of_the_clockwork_justiciar.initial_activation_delay = ark_time * 60`